docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -147,6 +151,46 @@ The API ML TLS requires servers to provide HTTPS ports. Each of the API ML servi
     - Authentication is service-dependent
     - Recommended to use the Authentication and Authorization Service for authentication
 
+#### Authentication endpoints
+
+The API Gateway contains two REST API authentication endpoints: `auth/login` and `auth/query`.
+
+The `/login` endpoint allows to authenticate mainframe user credentials and returns authentication token. The login request requires the user credentials in one of the following formats:
+  * Basic access authentication
+  * JSON body, which provides an object with the user credentials 
```

Comment:
Can we just put "JSON with user credentials"? or is "an object" important to mention here?